arcestia.json

```diff
@@ -0,0 +1,50 @@
+{
+  "basics": {
+    "name": "Laurensius Jeffrey Chandra",
+    "label": "Human Right Activist. Globalist and Investor.",
+    "image": "https://avatars1.githubusercontent.com/u/7936962?s=460&u=41e34bb816ad09323e1650f3efc0bec4fb2bc5dd&v=4",
+    "summary": "I’m now just a trader, investor who still like to code at freetime.",
```

Comment:
"summary": "I’m now just a trader, investor who still like to code at freetime.",

to

"summary": "I’m now just a trader and also an investor, who still like to code at free time.",

Reply:
wow a mistake spelling hmm, interesting